### PR TITLE
Add speedseq sv verbose argument.

### PR DESCRIPTION
**Pull request by jasonwalker80**

### CODE REVIEW

File: lib/perl/Genome/Model/Tools/Speedseq/Sv.pm [hunk 1]
```diff
@@ -110,8 +109,13 @@ class Genome::Model::Tools::Speedseq::Sv {
                     doc => 'output LUMPY probability curves in VCF',
                     is_optional => 1,
                     tool_param_name => 'P',
-                }
-
+                },
+                verbose => {
+                    is => 'Boolean',
+                    doc => 'verbose',
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Speedseq/Sv.pm]: This `doc` line is rather terse!

Comment (jasonwalker80) [inline lib/perl/Genome/Model/Tools/Speedseq/Sv.pm]: I agree.  I was merely using the description in the speedseq sv tool.  I updated the `doc` line a bit, but I'm not sure what else to include in the documentation.

Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Speedseq/Sv.pm]: Ah. Unless we have some special insight into what it does, I guess that works!

**Comment by tmooney** [comment]
+1 🔔

**Comment by acoffman** [comment]
+1

**Pull request merged**